tests/sles4sap/trento/deploy_trento.pm

```diff
@@ -20,13 +20,14 @@ sub run {
     my $machine_name = $self->get_vm_name;
     my $acr_name = $self->get_acr_name;
 
-    enter_cmd "cd /root/test";
+    my $cp = '/root/test';
+    enter_cmd "cd $cp";
 
+    my $sr = ' ./';
```

Comment:
Any reason to switch from `$deploy_script_run` to `$sr`? IMO it was clearer before.